modules/core/ecs-autoscaling-target/main.tf

```
@@ -4,8 +4,17 @@ resource "aws_appautoscaling_target" "ecs_target" {
   min_capacity       = var.min_capacity
   max_capacity       = var.max_capacity
   resource_id        = "service/${var.ecs_cluster_name}/${var.ecs_service_name}"
+  role_arn           = var.role_arn
   scalable_dimension = "ecs:service:DesiredCount"
   service_namespace  = "ecs"
+  dynamic "suspended_state" {
+    for_each = var.suspended_state == null ? [] : [var.suspended_state]
+    content {
```

nit: spacing for easier reading
```suggestion

    content {
```